youtube-client-app/src/app/components/filter/filter.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import {
+  ISort,
+  ISortSettings,
+  sortMap,
+  SortType,
+} from '../../common/models/sort-settings.model';
+
+@Component({
+  selector: 'app-filter',
+  templateUrl: './filter.component.html',
+  styleUrls: ['./filter.component.scss'],
+})
+export class FilterComponent {
+  @Output() public setSortSettings: EventEmitter<ISortSettings> = new EventEmitter();
+
+  public sortMap: ISort = sortMap;
+
+  private pressedSort: HTMLElement | undefined;
+
+  public sortSettings: ISortSettings = {
+    filterBy: sortMap.empty,
+    sortIncrease: true,
+    keywords: '',
+  };
+
+  constructor() {}
+
+  public changeSort(event: any, sortBy: SortType): void {
```

Comment:
it is almost never appropriate to use `any` type. it would be better if you replace it with something more concrete

Reply:
fixed